src/Repository/Phoning/CampaignRepository.php

```diff
@@ -47,4 +47,30 @@ public function findForAdherent(Adherent $adherent): array
             ->getResult()
         ;
     }
+
+    public function findPhoningCampaignsKpi()
+    {
+        return $this->createQueryBuilder('campaign')
+            ->select('COUNT(DISTINCT campaign.id) AS nb_campaigns')
+            ->addSelect('COUNT(DISTINCT IF(campaign.finishAt >= :end OR campaign.finishAt IS NULL , campaign.id, null)) AS nb_on_going_campaign')
+            ->addSelect('COUNT(campaignHistory.id) AS nb_calls')
+            ->addSelect('SUM(IF(campaignHistory.beginAt >= :start AND campaignHistory.beginAt <= :end, 1, 0)) AS nb_calls_last_month')
+            ->addSelect('SUM(IF(campaignHistory.dataSurvey IS NOT NULL, 1, 0)) as nb_surveys')
+            ->addSelect('SUM(IF(dataSurvey.postedAt >= :start AND dataSurvey.postedAt <= :end, 1, 0)) as nb_surveys_last_month')
+            ->leftJoin(
+                'campaign.campaignHistories',
+                'campaignHistory',
+                Join::WITH,
+                'campaignHistory.status != :send'
+            )
+            ->leftJoin('campaignHistory.dataSurvey', 'dataSurvey')
+            ->setParameters([
+                'end' => new \DateTime(),
+                'start' => new \DateTime('last month'),
```

Comment:
What about names corresponding to the date, to see immediately with what we compare?
```suggestion
                'now' => new \DateTime(),
                'last_month' => new \DateTime('last month'),
```